client/js/app/utils/ExplorerUtils.js

```diff
@@ -265,6 +265,10 @@ module.exports = {
       delete attrs['steps'];
     }
 
+    if (attrs.group_by) {
```

Comment:
is this `if` statement needed? `JSON.stringify(undefined)` is still undefined, and it doesn't look like `group_by` shows up in the URL when `attrs.group_by` is `undefined`.